controllers/imageclusterinstall_controller.go

```diff
@@ -1263,6 +1266,27 @@ func (r *ImageClusterInstallReconciler) writeInvokerCM(filePath string) error {
 	return nil
 }
 
+func (r *ImageClusterInstallReconciler) writeIBIOStartTimeCM(filePath string) error {
+	cm := &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: corev1.SchemeGroupVersion.String(),
+			Kind:       "ConfigMap",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: monitor.OcpConfigNamespace,
```

Comment:
Consider using `openshift-lifecycle-agent` NS
openshift-config has nothing to do with IBI